### PR TITLE
publish-python-package.yml addition 

### PR DESCRIPTION
**Pull request by lizlouise1335**
new publish yaml

### CODE REVIEW

**Comment by taylorfturner** [comment]
changed base to `feature/python_deploy`

File: .github/workflows/publish-python-package.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      run: |
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        if [ -f requirements-test.txt ]; then pip install -r requirements-test.txt; fi
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        TWINE_REPOSITORY: pypi
```
Comment (taylorfturner) [inline .github/workflows/publish-python-package.yml:31]: this will require some configs

Comment (taylorfturner) [reply .github/workflows/publish-python-package.yml:31]: otherwise LGTM

**Review by taylorfturner — CHANGES_REQUESTED** [review]
blocking until configs on backend are setup -- otherwise LGTM

**Review by taylorfturner — APPROVED** [review]
LGTM

File: .github/workflows/publish-python-package.yml
Comment (taylorfturner) [inline .github/workflows/publish-python-package.yml:1]: mimic of https://github.com/capitalone/DataProfiler/blob/main/.github/workflows/publish-python-package.yml almost

**Review by taylorfturner — APPROVED** [review]

**Review by danielbarcklow — APPROVED** [review]

**Pull request merged**